proto/batch_plan.proto

```diff
@@ -25,11 +25,10 @@ message RowSeqScanNode {
   // Whether the order on output columns should be preserved.
   bool ordered = 5;
 
-  message ChunkSize {
-    uint32 chunk_size = 1;
-  }
-  // If along with `batch_limit`, `chunk_size` will be set.
-  ChunkSize chunk_size = 6;
+  // deprecated
+  // ChunkSize chunk_size = 6;
+  // reserved 6;
+
```

Comment:
```suggestion
  message ChunkSize {
    uint32 chunk_size = 1;
  }
  ChunkSize chunk_size = 6 [deprecated = true];
```

Reply:
Sorry I should have left the suggestion directly 😆 I was referring to `deprecated` syntax of protobuf.